policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/pension_income_amount/cap.yaml

```diff
@@ -0,0 +1,11 @@
+description: British Columbia provides this maximum pension benefit.
```

Comment:
description: British Columbia provides this max pension benefit for eligible filers.